docs/integration/code-adaptions.mdx

```diff
@@ -0,0 +1,232 @@
+---
+sidebar_position: 3
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Adapting your game
+
+You should only need to make very few changes to your games to make them work well with the Arcade.
+
+Below we list the necessary changes and the SDK functions helping with them. In case you're using a language/framework that 
+we don't support yet, just reach out. All you need to create a new SDK (or work without one) is the ability to make a few HTTP calls.
+
+:::caution Attention
+Both the game server and client must respect errors.
+
+The Arcade SDK has built-in retry mechanisms, so if it returns an error or fails to initialize, then is error is final and
+the server and game client should disconnect and exit.
+:::
+
+:::note Game State
+Currently the Arcade doesn't support long-running games and therefore no state persistence is provided.
+If you're currently using some persistence, just replace it with some in-memory implementation.
+:::
+
+## Use a fixed random seed
+
+Probably your game has some kind of randomness to it: Where and when enemies spawn, how the level layout is like, etc.
+Now if two players play your game, there is a chance that one player has a significantly easier time than another player.
+Outside of the Arcade this is fine as it will average out over many games. 
+But for a highly competitive gameplay with money at stake this must not happen.
+
+Every player of the same leaderboard/challenge should get the same level, enemy behavior etc. 
+To achieve this you must explicitly set the seed of your random number generator with a seed that we provide to you.
+
+<Tabs groupId="rnd-seed">
+<TabItem value="unity" label="Unity">
+
+```cs
+private void AutoConnect_OnServerReady()
+{
+    UnityEngine.Random.InitState(AutoConnect.RandomSeed.GetHashCode());
+    // and we should only allow players to join after we setup all randomness
+}
+```
+
+</TabItem>
+<TabItem value="js" label="JavaScript">
+
+```js
+alert("TODO");
```

Comment:
@danthegoodman1 can you fill in the JS examples?

Reply:
Will do!